packages/devextreme/js/__internal/ui/chat/messageEditingPreview.ts

```diff
@@ -0,0 +1,133 @@
+import messageLocalization from '@js/common/core/localization/message';
+import type { dxElementWrapper } from '@js/core/renderer';
+import $ from '@js/core/renderer';
+import type { ClickEvent } from '@js/ui/button';
+import Button from '@js/ui/button';
+import type { Properties as DOMComponentProperties } from '@ts/core/widget/dom_component';
+import DOMComponent from '@ts/core/widget/dom_component';
+import type { OptionChanged } from '@ts/core/widget/types';
+
+export const CHAT_EDITING_PREVIEW_CLASS = 'dx-chat-editing-preview';
+export const CHAT_EDITING_QUOTE_LINE_CLASS = 'dx-chat-editing-quote-line';
```

Comment:
Let's not to render additional dom element and will add this line using :before() pseudo element